hook!-full-game/src/Player/Player.gd

```diff
@@ -1,5 +1,5 @@
 extends KinematicBody2D
-
+class_name Player
```

Comment:
If it's only for type checks, use e.g. `var Player = preload(...)` in your target script instead of registering the node globally